lib/Service/Logging.h

```diff
@@ -183,10 +208,15 @@
 
 #if (0 == LOG_DEBUG_ENABLE)
 
-#define LOG_DEBUG(_filename, _msg)           (void)(_filename)
+/** Log debug message. */
+#define LOG_DEBUG(_filename, _msg) (void)(_filename)
+/** Log debug message with additional value. */
 #define LOG_DEBUG_VAL(_filename, _msg, _val) (void)(_filename)
-#define LOG_DEBUG_HEAD(_filename)            (void)(_filename)
+/** Log debug header. */
+#define LOG_DEBUG_HEAD(_filename) (void)(_filename)

```

Comment:
Why we keep here (void)(_filename)?

Reply:
When DEBUG is not enabled, the TAGs in several modules will cause the compilation process to fail as these are not used.

Reply:
See https://github.com/BlueAndi/RadonUlzer/actions/runs/6810581334/job/18519155506

Reply:
Move the enable flags to the pio env's. Remove all LOGs from HAL + Service.